.gitignore

```diff
@@ -20,8 +20,9 @@ vendor
 # Tox environments
 .tox/
 *.dic
-/.vscode
+**/.vscode/
```

Comment:
do we need to update in this PR?

Reply:
May be a noob and or IDE user mistake but I am coding from the pkg dir in vscode. This produces the local .vscode file which gets picked up by git. I can remove it manually before committing but just made sense to add it now.

Reply:
if we need to change it we should do it in another PR, that asks for this change

Reply:
Done